velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -155,5 +155,66 @@ TEST_F(ProbabilityTest, binomialCDF) {
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
 
+TEST_F(ProbabilityTest, inverseBinomialCDF) {
+  const auto invBinomialCDF = [&](std::optional<int64_t> numberOfTrials,
+                                  std::optional<double> successProbability,
+                                  std::optional<double> p) {
+    return evaluateOnce<int64_t>(
+        "inverse_binomial_cdf(c0, c1, c2)",
+        numberOfTrials,
+        successProbability,
+        p);
+  };
+
+  EXPECT_EQ(0, invBinomialCDF(5, 0.5, 0.03125));
+  EXPECT_EQ(41, invBinomialCDF(41, 0.2, 1.0));
+  EXPECT_EQ(3, invBinomialCDF(5, 0.5, 0.8125));
+  EXPECT_EQ(3, invBinomialCDF(3, 0.8403, 0.5));
+  EXPECT_EQ(62, invBinomialCDF(200, 0.3, 0.6));
+  EXPECT_EQ(0, invBinomialCDF(79, 0.6, 0.0));
+  EXPECT_EQ(std::nullopt, invBinomialCDF(std::nullopt, 0.6, 0.0));
+  EXPECT_EQ(std::nullopt, invBinomialCDF(11, std::nullopt, 0.22));
+  EXPECT_EQ(std::nullopt, invBinomialCDF(134, 0.6, std::nullopt));
+  EXPECT_EQ(
+      std::nullopt, invBinomialCDF(std::nullopt, std::nullopt, std::nullopt));
+
+  // Invalid inputs for numberOfTrails
```

Comment:
As per coding guidelines https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md, comments should be full sentences that start with a capital letter and end with a full stop.

Please review the rest of the guidelines also.